01-main/packages/emptyepsilon

```diff
@@ -0,0 +1,10 @@
+DEFVER=1
+ARCHS_SUPPORTED="amd64"
+get_github_releases "daid/EmptyEpsilon" "latest"
+if [ "${ACTION}" != prettylist ]; then
+    URL="$(grep -m 1 "browser_download_url.*\.deb\"" "${CACHE_FILE}" | cut -d'"' -f4)"
+    VERSION_PUBLISHED="$(cut -d'-' -f2 <<< "${URL}" | cut -d'/' -f1)"
```

Comment:
This is OK.  However there may be an issue with how they handle their versioning:
Currently this ends up with
```
  Installed:	2024.12.8
  Published:	2024.12.08
```
which is actually probably OK as .09 will be seen as "higher" than .8, but the inconsistency is a little worrying...